packages/perseus/src/validation.types.ts

```diff
@@ -138,12 +141,12 @@ export type PerseusInteractiveGraphRubric = {
 
 export type PerseusInteractiveGraphUserInput = PerseusGraphType;
 
-/* TODO(LEMS-2440): Should be removed or refactored. Grading info may need
-    to be moved to the rubric from userInput. */
-export type PerseusLabelImageRubric = Empty;
+export type PerseusLabelImageScoringData = {
+    markers: ReadonlyArray<MarkerAnswers>;
+};
 
 export type PerseusLabelImageUserInput = {
-    markers: ReadonlyArray<InteractiveMarkerType>;
+    markers: ReadonlyArray<Omit<InteractiveMarkerType, "answers">>;
```

Comment:
These are the main changes. Changed `PerseusLabelImageUserInput` so it no longer includes `answers` in the array. Then I moved the parts of `InteractiveMarkerType` needed for scoring to PerseusLabelImageScoringData`.